modules/overview/pages/security-best-practices.adoc

```diff
@@ -60,10 +60,10 @@ All resources and actions you can grant users permission to use in their {namesp
 |"get", "list", "create"
 
 |configmaps
-|“get", "list", "create", "update", "patch", "delete"
+|"get", "list", "create", "update", "patch", "delete"
 
 |events
-|“watch”
+|"list", "watch"
```

Comment:
For `events`, the `list` permissions are also granted:
<img width="1186" alt="image" src="https://github.com/user-attachments/assets/5d154a4a-19b1-4515-9bf2-06f6a9eaf826" />